AUTHORS.rst

```diff
@@ -36,6 +36,8 @@ Contributors
 * Ben Thompson
 * Matthew Turk <matthewturk@gmail.com>
 * Miguel de Val-Borro <miguel.deval@gmail.com>
+* Jeroen Van Goey
+* Mike Zingale
```

Comment:
If @zingale prefers Mike, we should also update HISTORY.rst, where I used Michael

Reply:
no preference